dist/profile/manifests/catchall.pp

```diff
@@ -69,34 +69,33 @@
 
   # permalinks
   # - this one is referenced from 1.395.1 "sign post" release
-  Redirect /why            https://wiki.jenkins-ci.org/pages/viewpage.action?pageId=53608972
+  # TODO: Create a Jenkins Manifesto page?
```

Comment:
I'm not a big fan of committing TODOS
http://web.archive.org/web/20200511124337/https://wiki.jenkins.io/pages/viewpage.action?pageId=53608972 <-- its been deleted for a couple years now
http://web.archive.org/web/20170821200700/https://wiki.jenkins.io/pages/viewpage.action?pageId=53608972 <-- even before that, it was pretty dated

I think drop the todo and just redirect but no hard blocker.